packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1106,7 +1106,8 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 		}
 
 		function onPasteAsText() {
-			pasteAsPlainText(null);
+			const clipboardWithoutMarkdown = stripMarkup(MarkupToHtml.MARKUP_LANGUAGE_MARKDOWN, clipboard.readText());
+			pasteAsPlainText(clipboardWithoutMarkdown);
```

Comment:
Could you add a comment to explain what we are doing here? Maybe with a link to the issue being fixed. Ideally we should have unit tests for this (because we keep having regressions), but in the meantime a comment would help. It will prevent a future developer to remove your change to fix another bug.